src/index.html

```diff
@@ -13,6 +13,42 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__info">
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
+          alt="product"
+        />
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">Product code: 195434</p>
+      </div>
+      <div class="product-card__reviews">
+        <div class="product-card__reviews-stars">
+          <div class="product-card__star product-card__star--1--filled"></div>
```

Comment:
The class names 'product-card__star--1--filled' seem to indicate a modifier, but the double dashes are used incorrectly. If '1-filled' is meant to be a modifier, it should be 'product-card__star--1-filled'. If '1' is part of the element's name, it should be 'product-card__star-1-filled'.